src/platform/plugins/private/presentation_panel/public/panel_actions/edit_panel_action/edit_panel_action.ts

```diff
@@ -43,7 +43,7 @@ export class EditPanelAction
   public getDisplayName({ embeddable }: EmbeddableApiContext) {
     if (!isApiCompatible(embeddable)) throw new IncompatibleActionError();
     return i18n.translate('presentationPanel.action.editPanel.displayName', {
-      defaultMessage: 'Edit {value}',
+      defaultMessage: 'Edit {value} configuration',
```

Comment:
Change this to align with the `show_config_action` one when user has edit capabilities. Ideally both actions should be merged into either a single action or with a better controller in a follow up